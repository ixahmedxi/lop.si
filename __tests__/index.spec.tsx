import { render } from 'test-utils'
import Index from '../src/pages'

describe('Home Page', () => {
  it('matches snapshot', () => {
    const { container } = render(<Index />)
    expect(container).toMatchSnapshot()
  })
})
