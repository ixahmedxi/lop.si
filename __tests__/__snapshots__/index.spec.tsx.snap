// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home Page matches snapshot 1`] = `
.emotion-8 {
  padding-top: 48px;
}

.emotion-2 {
  box-sizing: border-box;
  margin: 0;
  min-width: 0;
  margin: 0 auto;
  width: 100%;
  text-align: center;
}

@media screen and (min-width:576px) {
  .emotion-2 {
    width: 100%;
  }
}

@media screen and (min-width:768px) {
  .emotion-2 {
    width: 100%;
  }
}

@media screen and (min-width:992px) {
  .emotion-2 {
    width: 70%;
  }
}

.emotion-0 {
  box-sizing: border-box;
  margin: 0;
  min-width: 0;
  color: var(--theme-ui-colors-primary,#8093f2);
  font-size: 20px;
  font-weight: 700;
  padding-bottom: 8px;
}

@media screen and (min-width:576px) {
  .emotion-0 {
    font-size: 20px;
    padding-bottom: 8px;
  }
}

@media screen and (min-width:768px) {
  .emotion-0 {
    font-size: 32px;
    padding-bottom: 16px;
  }
}

.emotion-1 {
  box-sizing: border-box;
  margin: 0;
  min-width: 0;
  line-height: 1.8;
  font-size: 12px;
}

@media screen and (min-width:576px) {
  .emotion-1 {
    font-size: 12px;
  }
}

@media screen and (min-width:768px) {
  .emotion-1 {
    font-size: 16px;
  }
}

.emotion-7 {
  box-sizing: border-box;
  margin: 0;
  min-width: 0;
  width: 100%;
  margin: 0 auto;
  padding-top: 24px;
}

@media screen and (min-width:576px) {
  .emotion-7 {
    width: 100%;
    padding-top: 24px;
  }
}

@media screen and (min-width:768px) {
  .emotion-7 {
    width: 100%;
    padding-top: 40px;
  }
}

@media screen and (min-width:992px) {
  .emotion-7 {
    width: 70%;
  }
}

.emotion-6 {
  display: grid;
  grid-template-columns: auto 1fr auto;
  place-items: center;
  border-radius: 100px;
  padding: 4px;
  border: 3px solid;
  border-color: transparent;
  -webkit-transition: border-color 0.2s ease-out;
  transition: border-color 0.2s ease-out;
  box-shadow: none;
}

.emotion-3 {
  font-size: 12px;
  opacity: 0.5;
  margin-left: 24px;
}

@media screen and (min-width:576px) {
  .emotion-3 {
    font-size: 12px;
    margin-left: 24px;
  }
}

@media screen and (min-width:768px) {
  .emotion-3 {
    font-size: 16px;
    margin-left: 40px;
  }
}

.emotion-4 {
  box-sizing: border-box;
  margin: 0;
  min-width: 0;
  display: block;
  width: 100%;
  padding: 16px;
  -webkit-appearance: none;
  -moz-appearance: none;
  appearance: none;
  font-size: inherit;
  line-height: inherit;
  border: 1px solid;
  border-radius: 4px;
  color: inherit;
  background-color: transparent;
  outline: none;
  border: none;
  padding-left: 24px;
  padding-right: 24px;
  height: 40px;
  font-size: 12px;
}

@media screen and (min-width:576px) {
  .emotion-4 {
    height: 40px;
    font-size: 12px;
  }
}

@media screen and (min-width:768px) {
  .emotion-4 {
    height: 48px;
    font-size: 16px;
  }
}

.emotion-4::-webkit-input-placeholder {
  color: var(--theme-ui-colors-text,#EEEEEE);
  opacity: 0.5;
}

.emotion-4::-moz-placeholder {
  color: var(--theme-ui-colors-text,#EEEEEE);
  opacity: 0.5;
}

.emotion-4:-ms-input-placeholder {
  color: var(--theme-ui-colors-text,#EEEEEE);
  opacity: 0.5;
}

.emotion-4::placeholder {
  color: var(--theme-ui-colors-text,#EEEEEE);
  opacity: 0.5;
}

.emotion-5 {
  height: 40px;
  width: 40px;
  outline: none;
  border: none;
  font-size: 16px;
  background-color: var(--theme-ui-colors-primary,#8093f2);
  color: var(--theme-ui-colors-background,#2B2D32);
  border-radius: 100px;
  padding: 0;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  cursor: pointer;
  box-shadow: none;
}

@media screen and (min-width:576px) {
  .emotion-5 {
    height: 40px;
    width: 40px;
    font-size: 16px;
  }
}

@media screen and (min-width:768px) {
  .emotion-5 {
    height: 48px;
    width: 48px;
    font-size: 20px;
  }
}

<div>
  <div
    class="emotion-8"
  >
    <div
      class="emotion-2"
    >
      <h1
        class="emotion-0"
      >
        Create professional looking urls
      </h1>
      <p
        class="emotion-1"
      >
        lop.si helps you transform long and unmanageable urls into short alternatives that look more professional wherever used.
      </p>
    </div>
    <div
      class="emotion-7"
    >
      <form
        autocomplete="off"
        class="emotion-6"
        novalidate=""
      >
        <svg
          class="emotion-3"
          fill="none"
          height="1em"
          stroke="currentColor"
          stroke-linecap="round"
          stroke-linejoin="round"
          stroke-width="2"
          viewBox="0 0 24 24"
          width="1em"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            d="M21.44 11.05l-9.19 9.19a6 6 0 0 1-8.49-8.49l9.19-9.19a4 4 0 0 1 5.66 5.66l-9.2 9.19a2 2 0 0 1-2.83-2.83l8.49-8.48"
          />
        </svg>
        <input
          class="emotion-4"
          name="url"
          placeholder="Paste your url here..."
          type="url"
        />
        <button
          aria-label="create a short url"
          class="emotion-5"
          type="submit"
        >
          <svg
            fill="none"
            height="1em"
            stroke="currentColor"
            stroke-linecap="round"
            stroke-linejoin="round"
            stroke-width="2"
            viewBox="0 0 24 24"
            width="1em"
            xmlns="http://www.w3.org/2000/svg"
          >
            <line
              x1="5"
              x2="19"
              y1="12"
              y2="12"
            />
            <polyline
              points="12 5 19 12 12 19"
            />
          </svg>
        </button>
      </form>
    </div>
  </div>
</div>
`;
